R/parse-body.R

```diff
@@ -519,6 +519,8 @@ parser_multi <- function() {
     if (!stri_detect_fixed(content_type, "boundary=", case_insensitive = TRUE))
       stop("No boundary found in multipart content-type header: ", content_type)
     boundary <- stri_match_first_regex(content_type, "boundary=([^; ]{2,})", case_insensitive = TRUE)[,2]
+    # Remove surrounding quotes if they exist
+    boundary <- stri_replace_first_regex(boundary, '^"(.*)"$', "$1")
```

Comment:
```suggestion
    # Remove surrounding quotes if they exist
    boundary <- stri_match_first_regex(content_type, "boundary=\"?([^; \"]{2,})\"?", case_insensitive = TRUE)[,2]
```

Reply:
@schloerke Do you want to enforce the 70 character delimiter limit?

...{2,70}?